cmd/stencil/stencil.go

```diff
@@ -40,7 +40,7 @@ func main() {
 	app := cli.App{
 		Version:     version.Version,
 		Name:        "stencil",
-		Description: "a smart templating engine for service development",
+		Description: "a smart templating engine for project development",
```

Comment:
I'll be marketing this more as a repository management tool, but for now this works.

Reply:
yeah, i was torn between repo and project, but given that it can be at any level in a repo, and you may even want multiple stencils per repo in some weird case, i figured project made more sense as terminology.  dunno how you're thinking about it in that sense

Reply:
That's a fair point, haven't considered the use case of being used in sub directories of a repository. Given that we expose information about a repository, I'm more inclined to generally target whole repository scenarios. But, I don't think we'd ever try to break being used multiple times.